### PR TITLE
Registry code cleanup for Locust backend

### PR DESCRIPTION
**Pull request by diegomarangoni**
EES-4629

This PR removes all unnecessary code after the complete migration to registry pattern related to Locust backend.

### CODE REVIEW

**Review by boekkooi-fresh — COMMENTED** [review]
Why did you not just remove `pkg/backends/locust/locust.go`?

File: pkg/backends/locust/backend.go [hunk 1]
```diff
@@ -15,11 +16,23 @@ import (
...
 )
 
+var (
+	defaultImageName = "locustio/locust"
+	defaultImageTag  = "latest"
+)
+
+var (
+	// ErrRequireMinOneDistributedPod spec requires 1 or more DistributedPods
+	ErrRequireMinOneDistributedPod = errors.New("LoadTest must specify 1 or more DistributedPods")
```
Comment (vgarvardt) [inline pkg/backends/locust/backend.go:26]:
```suggestion
	ErrRequireMinOneDistributedPod = errors.New("LoadTest must specify at least 1 DistributedPods")
```

File: pkg/backends/locust/backend_test.go [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+	assert.NotEmpty(t, configMaps.Items, "Expected non-zero configMaps amount after CheckOrCreateResources but found zero")
+}
+
+func TestSyncStatus(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Fake clients
+	kubeClient := k8sfake.NewSimpleClientset()
+	logger, _ := zap.NewDevelopment()
```
Comment (vgarvardt) [inline pkg/backends/locust/backend_test.go:70]: zaptest pls

File: pkg/backends/locust/backend.go (hunk 1 above)
Comment (diegomarangoni) [reply pkg/backends/locust/backend.go:26]: Changing this here is out of scope, this is an existing error

**Comment by s4nji** [comment]
+1

**Pull request merged**